catalyst_voices/packages/catalyst_voices_shared/lib/src/platform_widget_builder/platform_widget_builder.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:catalyst_voices_shared/src/platform/catalyst_platform.dart';
+import 'package:flutter/widgets.dart';
+
+class PlatformWidgetBuilder<T> extends StatelessWidget {
```

Comment:
Why do we need a Generic here ?
Are we suppose to use something else except `String` type here ?

Reply:
Yes, this is made on purpose. The widget is just an abstraction to integrate the platform detection and it will always be used to wrap other widgets that will consume the platform specific value.

Two simple examples:
- passing a `String` as you mentioned to visualize customized the text visualized based on the platform
- passing a `Size` to the wrapped widget (i.e. on native iOS app you have some elements in place that need specific spacing)